Localization/Translations/Options/Sounds.lua

```diff
@@ -170,6 +170,30 @@ local soundsOptionsLocales = {
         ["esES"] = "El sonido que escuchas cuando avanzas en un objetivo de misión",
         ["frFR"] = "Le son que vous entendez lorsque vous progressez sur un objectif de quête",
     },
+    ["Sound Delay"] = {
+        ["ptBR"] = false,
+        ["ruRU"] = false,
+        ["deDE"] = "Sound-Verzögerung",
+        ["koKR"] = false,
+        ["esMX"] = false,
+        ["enUS"] = true,
+        ["zhCN"] = false,
+        ["zhTW"] = false,
+        ["esES"] = false,
+        ["frFR"] = false,
+    },
+    ["Delay (in seconds) between the event and playing the sound."] = {
```

Comment:
I think it would be good to explain this setting with some more details. It is only for progress events and should only be changed on "good" hardware. Once users hear double sounds, they need to increase it.